docs/api/tutorials/start.md

```diff
@@ -0,0 +1,206 @@
+
+In Red, each table is represented using a special type of class called "model". It can do
+everything what a usual class can do, but also helps you to interact with your table.
+
+Red models use `model` keyword instead of `class`:
+
+```perl6
+model Person {
+    has Int $.id is serial;
+    has Str $.name is column is rw;
+}
+
+my $*RED-DB = database “SQLite”;
+```
+
+We described a model called `Person`. The first attribute `$.id` is typed to be `Int`
+and is marked with `is serial` trait. This trait marks the column as a primary one with
+autoincrement enabled. The next attribute `$.name` is marked with `is column` trait, which
+means this attribute will be mapped onto a column in the table, and is typed as Str.
+
+Note we don't need to specify that the column is not nullable, as it is the default.
+
+The second statement specifies a database to work with. In this case,
+an in-memory SQLite database is used, which means all changes will be lost after
+the script termination. To avoid this, we can specify a name for the database file:
+
+```perl6
+my $*RED-DB = database “SQLite”, database => 'test.sqlite3'; # Now a file `test.sqlite3` will be created
+```
+
+Next, we need to create a table itself:
+
+
+```perl6
+Person.^create-table;
+```
+
+Methods marked with `^` are called "meta methods" and are used in Red
+for all kinds of operations on models. In this case, calling `^create-table`
+creates a table with name `person`.
+
+### Insertion of new records
+
+Let's insert a new record into it. In SQL it could be:
+
+```sql
+INSERT INTO person (name) VALUES 'John';
+```
+
+In Red we can express it this way:
+
+```perl6
+my $person = Person.^create(name => 'John');
+```
+
+We call the `^create` method on type object `Person` and assign the result
+to the `$person` variable. The assignment is not necessary:
+
+```perl6
+Person.^create(name => 'John');
+```
+
+The `^create` method returns the created object to work with, though
+this result can be simply ignored.
+
+The `$.id` attribute is auto-generated and there is no need to specify it,
+while `$.name` attribute must not be null, so we have to specify it:
+
+```perl6
+Person.^create; # error
+```
+
+### Update of records
+
+Let's try to update our record. In SQL it could be:
+
+```sql
+UPDATE person SET name = ‘John Doe’ WHERE id = 1;
+```
+
+To do the same in Red, we use setters and a call to `^save`:
+
+```perl6
+$person.name = 'John Doe';
+$person.^save;
+```
+
+All changes to an object that represents the record are lazy,
+which means the database connection is not used until the `^save`
+method is called.
+
+The method `^save` is useful not only for UPDATE operation, but it can be used on
+INSERT too:
+
+```perl6
+my $person2 = Person.new(name => 'Joan'); # ^create is not used
+$person2.^save; # does INSERT, not UPDATE
+```
+
+### Selecting records
+
+Lets add some more records:
+
+```perl6
+Person.new(name => "Paul"); # Method call with parentheses and an arrow pair
```

Comment:
This to lines are not inserting on the database... they are just creating objects. It should call `.^save` or have used `.^create` to store on the database

Reply:
Ah, nice catch!